jenkins/scality-glance-store-functional-tests/extras.d/55-scality-glance-store.sh

```diff
@@ -0,0 +1,28 @@
+# 55-scality-glance-store - Devstack extra script to configure Glance API with Scality store
+
+function install_scality_store {
+    sudo pip install https://github.com/scality/scality-sproxyd-client/archive/master.tar.gz
+    # For some reason, doing this failed: glance-api would not restart,
+    # complaining about an old version of python-six.
+    # Probably a setuptools version issue
+    # sudo python setup.py install
+    sudo pip install https://github.com/scality/scality-glance-store/archive/master.tar.gz
```

Comment:
So far we only have and test master. I don't see use maintaining several versions of this in the neat future. When needed, I agree, this would have to be changed.